frontend/src/App.vue

```diff
@@ -124,29 +124,17 @@ export default {
   watch: {
     "prompts": {
       handler(promptsVal) {
-        if (_.isEmpty(promptsVal)) {
-          this.showBusyIndicator = true
-        } else {
-          this.showBusyIndicator = false
-        }
+        this.setBusyIndicator(_.isEmpty(promptsVal))
```

Comment:
I would call this.setBusyIndicator() without params, and let the func to calculate isBussy.
1 flow can set busy to false while other flow is still in busy - true.